core/zoe/contracts/simpleOffer/simpleOffer.js

```diff
@@ -0,0 +1,54 @@
+import harden from '@agoric/harden';
+
+import makePromise from '../../../../util/makePromise';
+
+const makeSimpleOfferMaker = srcs => zoeInstance => {
```

Comment:
The layering here needs more documentation. If SWAPS.md is intended to document the use of this file, then there should be a link here.

Please describe the expectations of isValidOffer(), reallocate(), and canReallocate(). 

If this hides zoe from the contracts above it, then it should say more about how they interact.

Is the use of simpleOffer apparent to the client of a second price auction? What do they see?